CONTRIBUTING.md

```diff
@@ -140,6 +140,14 @@ Instead, you need to say
 bundle exec appraisal 5.1 rspec spec/acceptance/active_model_integration_spec.rb
 ```
 
+### Run all tests
+
+In order to run all tests, simply run
```

Comment:
Colon at the end of this sentence? (I know there isn't one at the end of line 137, but that was a mistake.)